colabs/huggingface/wandb_hf_example.ipynb

```diff
@@ -0,0 +1,12795 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    os.environ['WANDB_WATCH'] = "all" # log of "gradients" and "parameters" of the model
</pre><p>@Morgan Do we still want to include watch in our examples?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/355/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
I remember @morganmcg1 mentioning that we shouldn't try to show this off as something we suggest they turn on by default due to the noticeable slowdown in training

Reply:
Thanks @ash0ts . The recent PR I raised with transformers, [changed the default setting](https://github.com/huggingface/transformers/blob/b3a0aad37dee31cddc532ccbb63600458a6b8419/src/transformers/integrations.py#L690) to "false". It was "gradients" earlier and that was causing the slowdown in training. Now, it's up to the user to turn it on. Since this is a demo. I'm using "all" to showcase the feature completely.


Will defer to @morganmcg1's comment on this though.

Reply:
<p>+1 lets remove mentions of watch</p>